src/main/resources/static/templates/SessionManagementVulnerability/Common/SessionManagement.css

```
@@ -0,0 +1,116 @@
+}
+
+.session-response {
+    margin: 0;
+    padding: 8px;
+    border: 1px solid black;
+    border-radius: 4px;
+    background: #fff;
+    color: black;
+    text-align: left;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)